ref_ts/migrations/0001_initial.py

```diff
@@ -1,119 +0,0 @@
-                ('user', models.ForeignKey(related_name='reftimeseriess', verbose_name='Author', to=settings.AUTH_USER_MODEL)),
-                ('view_groups', models.ManyToManyField(help_text=b'This is the set of Hydroshare Groups who can view the resource', related_name='group_viewable_ref_ts_reftimeseries', null=True, to='auth.Group', blank=True)),
-                ('view_users', models.ManyToManyField(help_text=b'This is the set of Hydroshare Users who can view the resource', related_name='user_viewable_ref_ts_reftimeseries', null=True, to=settings.AUTH_USER_MODEL, blank=True)),
-            ],
-            options={
-                'ordering': ('_order',),
-                'verbose_name': 'Referenced HIS Time Series Resource',
-            },
-            bases=('pages.page', models.Model),
-        ),
-        migrations.CreateModel(
-            name='RefTSMetadata',
-            fields=[
-                ('coremetadata_ptr', models.OneToOneField(parent_link=True, auto_created=True, primary_key=True, serialize=False, to='hs_core.CoreMetaData')),
-            ],
-            options={
-            },
-            bases=('hs_core.coremetadata',),
-        ),
-        migrations.CreateModel(
-            name='Site',
-            fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('object_id', models.PositiveIntegerField()),
-                ('name', models.CharField(max_length=100)),
-                ('code', models.CharField(max_length=50)),
-                ('latitude', models.DecimalField(null=True, max_digits=9, decimal_places=6)),
-                ('longitude', models.DecimalField(null=True, max_digits=9, decimal_places=6)),
-                ('content_type', models.ForeignKey(related_name='ref_ts_site_related', to='contenttypes.ContentType')),
-            ],
-            options={
-                'abstract': False,
-            },
-            bases=(models.Model,),
-        ),
-        migrations.CreateModel(
-            name='Variable',
-            fields=[
-                ('id', models.AutoField(verbose_name='ID', serialize=False, auto_created=True, primary_key=True)),
-                ('object_id', models.PositiveIntegerField()),
-                ('name', models.CharField(max_length=100)),
-                ('code', models.CharField(max_length=50)),
-                ('data_type', models.CharField(max_length=50, null=True)),
-                ('sample_medium', models.CharField(max_length=50, null=True)),
-                ('content_type', models.ForeignKey(related_name='ref_ts_variable_related', to='contenttypes.ContentType')),
-            ],
-            options={
-                'abstract': False,
-            },
-            bases=(models.Model,),
-        ),
-    ]
```

Comment:
As with the others, any removed or modified migration must be addressed or explained.